dds_web/api/schemas/user_schemas.py

```diff
@@ -143,11 +144,12 @@ class NewUserSchema(marshmallow.Schema):
     )
     password = marshmallow.fields.String(
         required=True,
-        validate=marshmallow.validate.Regexp(
-            regex="^(?=[^a-z\n]*[a-z])(?=[^A-Z\n]*[A-Z])(?=[^\d\n]*\d)^[\S]{10,64}$",
-            # Password must contain one or more uppercase, lowercase and digit characters.
-            # It may contain special characters (not enforced), as they might be hard to type depending on keyboard layout (e.g. on smartphones)
-            # Length of the password must be between 10 and 64 characers
+        validate=marshmallow.validate.And(
```

Comment:
If you want to painstakingly adhere to the KRT 332 technical requirements document, the condition should read
` Length AND lowercase AND uppercase AND (digit OR special character)`

Reply:
Yep, it's changed now 👍🏻